include/boost/beast/core/static_buffer.hpp

```diff
@@ -46,18 +60,98 @@ class static_buffer_base
     std::size_t out_size_ = 0;
     std::size_t capacity_;
 
+    class const_buffer_pair;
+
+    class mutable_buffer_pair
+    {
+        boost::asio::mutable_buffer b_[2];
+
+        friend class const_buffer_pair;
+
+    public:
+        using const_iterator =
+            boost::asio::mutable_buffer const*;
+
+        // workaround for buffers_iterator bug
+        using value_type =
+            boost::asio::mutable_buffer;
+
+        mutable_buffer_pair() = default;
+        mutable_buffer_pair(
+            mutable_buffer_pair const&) = default;
+
+        boost::asio::mutable_buffer&
+        operator[](int i) noexcept
```

Comment:
And these are also questionably noexcept.

Reply:
Why wouldn't it be `noexcept`?